website/content/en/applications.adoc

```diff
@@ -3,41 +3,57 @@ title: "About Applications for FreeBSD"
 sidenav: about
 ---
 
-= About Applications for FreeBSD
+= Applications for FreeBSD
 
-== Experience the possibilities with FreeBSD
+FreeBSD goes beyond what's commonly expected of a UNIX(R) workstation.
```

Comment:
Contractions? I do not know if those are usual in the documentation, and "what is" might be more appropriate here.

Reply:
Thanks, does [`6bb7fe0` (#103)](https://github.com/freebsd/freebsd-doc/pull/103/commits/6bb7fe0305069ed3f9a120492f2ed7a213a789f2)# resolve this you? If not please un-resolve. 

True: contractions are not the norm, however some parts of the current edition <https://web.archive.org/web/20221025103015/https://www.freebsd.org/applications/> have a colloquial feel to them … I did think about (non-contracted) _what is_ whilst drafting, but it felt too non-colloquial at the time.